src/main/java/fr/sncf/osrd/railjson/parser/RailJSONParser.java

```diff
@@ -68,9 +68,9 @@ public static Infra parse(RJSInfra railJSON) throws InvalidInfraException {
         var switchNames = new HashMap<String, Switch>();
         var switchIndex = 0;
         for (var rjsSwitch : railJSON.switches) {
-            var index = nodeIDs.get(rjsSwitch.base);
+            var index = nodeIDs.get(rjsSwitch.getBase());
```

Comment:
Remove the function `getBase` since it's use only here. Comment why it's working.